### DISCUSSION TITLE
Does "fates_recruit_height_min" affect allometry?

### DISCUSSION THREAD

**Discussion by adamhb** [NGEET/fates#949] [Q&A] [answered]
Hi Fates Team,
The parameter "fates_recruit_height_min" seems to be quite a sensitive parameter and I've been trying to figure of out if it affects allometry in any way beyond determining the dbh of newly spawned cohorts and thereby affecting the other biomass pools of newly recruited cohorts. If anyone is aware of it doing anything besides the above please let me know!
cheers,
Adam

**Comment by JoshuaRady** [comment, marked as answer]
Following up on discussion in today's FATES teleconference (1/5/2023), `fates_recruit_height_min` does not effect allometry in FATES.  However, note that the default tree PFTs currently have values of `fates_recruit_height_min` = 1.3, around breast height.  Values shorter than this could lead to DBH values of 0 or below for some allometries, which would likely cause problems or at least strange behavior.  Check your allometry prior to decreasing this value.  Beyond that warning there is nothing magic about `fates_recruit_height_min`.

**Reply by adamhb** [reply]
Thanks @JoshuaRady! This is consistent with my understanding too.
Adam
